pkg/tracking/jira/tickets.go

```diff
@@ -16,16 +19,44 @@ func (tc TC) GetTicket(id string) (*model.Ticket, error) {
 	if err != nil {
 		return nil, err
 	}
+	ticket.URLTracker = fmt.Sprintf("%s/browse/%s", tc.URL, ticket.Key)
+
 	return ticket, nil
 }
 
-// CreateTicket creates an ticket in Jira.
+// FindTicketByFindingAndTeam retrieves a ticket from Jira using the project key, the vulnerability issue type,
+// the finding ID and the team ID.
+// Return an empty ticket if not found.
+func (tc TC) FindTicketByFindingAndTeam(projectKey, vulnerabilityIssueType, findingID string, teamID string) (*model.Ticket, error) {
+	text := fmt.Sprintf("\"%s\"", common.GenerateIdentificationText(findingID, teamID))
+
+	ticket, err := tc.Client.FindTicket(projectKey, vulnerabilityIssueType, text)
+	if err != nil {
+		return nil, err
+	}
+	return ticket, nil
+}
+
+// CreateTicket creates a ticket in Jira.
 func (tc TC) CreateTicket(ticket *model.Ticket) (*model.Ticket, error) {
+
+	ticketInJira, err := tc.FindTicketByFindingAndTeam(ticket.Project, ticket.TicketType, ticket.FindingID, ticket.TeamID)
```

Comment:
I think we must check the returned `err` in this call before checking if the value of this variable is `ticketInJira` is nil.

Reply:
Fixed in commit bf2bad44a135a34343bce16ad073046b5ca71723